### ISSUE TITLE
Compile Error 4.15.3, when trying to setup cross compile

### ISSUE THREAD

**Issue by aspartamed** [Koderz/RuntimeMeshComponent#50]
I get the following build error while building on 

- Windows 10
- Visual Studio 2015
- v8_clang-3.9.0-centos7
Github check out has last commit "Branch to sanpshot for CL 3451409" on May 22 by unreal bot

```
25>------ Build started: Project: TestPAL, Configuration: Linux_Development_Program Win32 ------
12>  Using LINUX_MULTIARCH_ROOT, building with toolchain 'D:\dev\Linux_CorssCompileToolchain\v8_clang-3.9.0-centos7\x86_64-unknown-linux-gnu'
12>  Target is up to date
12>  Deploying UE4Game Linux Development...
12>  Total build time: 0.83 seconds (NoActionsToExecute executor: 0.00 seconds)
13>  Using LINUX_MULTIARCH_ROOT, building with toolchain 'D:\dev\Linux_CorssCompileToolchain\v8_clang-3.9.0-centos7\x86_64-unknown-linux-gnu'
13>  Target is up to date
13>  Deploying BlankProgram Linux Development...
13>  Total build time: 0.15 seconds (NoActionsToExecute executor: 0.00 seconds)
14>  Using LINUX_MULTIARCH_ROOT, building with toolchain 'D:\dev\Linux_CorssCompileToolchain\v8_clang-3.9.0-centos7\x86_64-unknown-linux-gnu'
14>  Target is up to date
14>  Deploying BuildPatchTool Linux Development...
14>  Total build time: 0.21 seconds (NoActionsToExecute executor: 0.00 seconds)
16>  Using LINUX_MULTIARCH_ROOT, building with toolchain 'D:\dev\Linux_CorssCompileToolchain\v8_clang-3.9.0-centos7\x86_64-unknown-linux-gnu'
16>  Performing 1 actions (4 in parallel)
16>  [1/1] Link CrossCompilerTool
16>  D:/dev/UnrealEngineSource/UnrealEngine/Engine/Intermediate/Build/Linux/B4D820EA/CrossCompilerTool/Development/ShaderFormatOpenGL/Module.ShaderFormatOpenGL.cpp.o: In function `FOpenGLFrontend::PrecompileShader(FShaderCompilerOutput&, FShaderCompilerInput const&, char const*, GLSLVersion, EHlslShaderFrequency)':
16>  D:\dev\UnrealEngineSource\UnrealEngine\Engine\Source\Developer\ShaderFormatOpenGL\Private/OpenGLShaderCompiler.cpp:1158: undefined reference to `glCreateShader'
16>  D:\dev\UnrealEngineSource\UnrealEngine\Engine\Source\Developer\ShaderFormatOpenGL\Private/OpenGLShaderCompiler.cpp:1161: undefined reference to `glShaderSource'
16>  D:\dev\UnrealEngineSource\UnrealEngine\Engine\Source\Developer\ShaderFormatOpenGL\Private/OpenGLShaderCompiler.cpp:1165: undefined reference to `glCompileShader'
16>  D:\dev\UnrealEngineSource\UnrealEngine\Engine\Source\Developer\ShaderFormatOpenGL\Private/OpenGLShaderCompiler.cpp:1168: undefined reference to `glGetShaderiv'
16>  D:\dev\UnrealEngineSource\UnrealEngine\Engine\Source\Developer\ShaderFormatOpenGL\Private/OpenGLShaderCompiler.cpp:1183: undefined reference to `glGetShaderiv'
16>  D:\dev\UnrealEngineSource\UnrealEngine\Engine\Source\Developer\ShaderFormatOpenGL\Private/OpenGLShaderCompiler.cpp:1192: undefined reference to `glGetShaderInfoLog'
16>  D:\dev\UnrealEngineSource\UnrealEngine\Engine\Source\Developer\ShaderFormatOpenGL\Private/OpenGLShaderCompiler.cpp:1220: undefined reference to `glDeleteShader'
16>clang++.exe : error : linker command failed with exit code 1 (use -v to see invocation)
16>ERROR : UBT error : Failed to produce item: D:\dev\UnrealEngineSource\UnrealEngine\Engine\Binaries\Linux\CrossCompilerTool
16>  Total build time: 3.19 seconds (Local executor: 0.00 seconds)
```

**Comment by Koderz** [comment]
Hello,

All existing issues for RMCv2 are being closed as RMCv2 is no longer being supported.  RMCv3 is in beta here : https://github.com/Koderz/RuntimeMeshComponent/tree/Version3Dev 

If you encounter this or any new issues in v3 please feel free to open a new issue! 

Thanks!

-Koderz
